src/neondis.cpp

```diff
@@ -0,0 +1,41 @@
+#include <fstream>
+#include <iostream>
+#include <sstream>
+
+#include "disassembler.h"
+
+int main(int argc, char *argv[])
+{
+    if (argc < 2) {
+        fprintf(stderr, "Usage: %s file1.neonx file2.neonx...fileN.neonx\n", argv[0]);
+        exit(1);
+    }
+
+    int a = 1;
+    while (a < argc) {
+        std::cout << "Disassembeling " << argv[a] << "...\n";
```

Comment:
You don't need to print a status message if the thing you're about to do writes to stdout anyway. Remove this.